### PR TITLE
ODC-7484,ODC-7485,ODC-7486: Added Task, TaskRun and ClusterTask list pages to dynamic plugin

### PR DESCRIPTION
**Pull request by lokanandaprabhu**
**Story:**
https://issues.redhat.com/browse/ODC-7484
https://issues.redhat.com/browse/ODC-7485
https://issues.redhat.com/browse/ODC-7486

**Description:**
Added Task, TaskRun and ClusterTask list pages to dynamic plugin

### CODE REVIEW

File: src/components/pipelines-tasks/ClusterTaskRow.tsx [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+    <DropdownItem
+      key={KEBAB_ACTION_DELETE_ID}
+      component="button"
+      onClick={launchDeleteModal}
+      isDisabled={!canDeleteCronTab[0]}
+      data-test-action={KEBAB_ACTION_DELETE_ID}
+    >
+      {t('Delete {{resourceKind}}', { resourceKind: ClusterTaskModel.kind })}
+    </DropdownItem>,
+  ];
```
Comment (vikram-raj) [inline src/components/pipelines-tasks/ClusterTaskRow.tsx:114]: Can you create a common kebab component called `K8sCommonKebabMenu` and use it? This component will have Edit annotation, Edit labels, Edit Resource(which will redirect to yaml page of the resource), and Delete options.

**Comment by lokanandaprabhu** [comment]
Updated the PR based on review comments.

**Comment by ppitonak** [comment]
Can you please explain me how it would work for end user after we merge this?
1. Will OpenShift Console 4.16 still contain these pages? What about further releases?
2. What happens when I enable dynamic plugin on my cluster? Will console always show the version from the dynamic plugin instead of static one?

**Comment by lokanandaprabhu** [comment]
/close

Handled in PR - https://github.com/openshift-pipelines/console-plugin/pull/78

**Pull request closed without merging**